akd/src/tree_node.rs

```diff
@@ -429,10 +436,17 @@ impl TreeNode {
         new_leaf: Self,
         epoch: u64,
         num_nodes: &mut u64,
-        include_ep: Option<bool>,
+        leaf_hash_mode: HashMode,
     ) -> Result<(), AkdError> {
-        self.insert_single_leaf_helper::<_>(storage, new_leaf, epoch, num_nodes, false, include_ep)
```

Comment:
Same as above

Reply:
One of the main ideas of oZKS is to include the epoch in the leaf (hash) so that we have a proof of when the leaf was inserted. Looks like a naming error to me.